### ISSUE TITLE
if Ops are on People cant use crates

### ISSUE THREAD

**Issue by marioandluigi878** [ConnorLinfoot/CratesPlus#26]
for some reason only ops can use crates and keys is there a permission im missing or whats going on 
cause if im missing a permission please make it clear on the spigot site for the plugin

**Comment by marioandluigi878** [comment]
let me rephrase that 
whenever an op is on pepole cant use crates

**Comment by ConnorLinfoot** [comment]
There is no permissions for opening crates. This will be an issue on your server, not the plugin.
